src/main/resources/logback.xml

```diff
@@ -8,6 +8,13 @@
         </filter>
     </appender>
 
+    <appender name="HEALTHCHECK-FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/var/logs/dks/healthcheck.log</file>
+        <append>true</append>
+        <encoder>
+            <pattern>[%d{yyyy-MM-dd HH:mm:ss}] [%level] [%file:%method:%line] %message%n</pattern>
+        </encoder>
+    </appender>
```

Comment:
Do we want to roll the file over maybe, and have a deletion policy? (We are going to introduce something similar for the main log).

Reply:
Maybe we can add this to both logs in the same ticket? If you already have a plan then it is probably a good idea to keep them separate?